app/controllers/jira/status/push_controller.rb

```diff
@@ -26,12 +28,24 @@ class PushController < ApplicationController
           JiraIssuesAndPushes::ERROR_BLANK_LONG_RUNNING_MIGRATION.to_s => 'Migrations field is blank'
         }
       }.freeze
-
       before_action :find_sha_resources,    except: [:branch, :summary]
       before_action :find_branch_resources, only:   [:branch]
 
       def edit; end
 
+      def deploy_email
+        text =
+          if @push.email_sent?
+            'Email was already sent.'
+          else
+            DeployMailer.deployment_email(@push.jira_issues).deliver_now!
+            @push.update(email_sent: true)
+            @push.save
+            'Email has been sent.'
```

Comment:
Same here, would be nice to echo on the page what we sent in the email.. 

so either redirect to the existing 'edit' page, or could we just render the table [that's put in the email] in this view too?